lib/active_model/serializer/utils.rb

```diff
@@ -0,0 +1,24 @@
+module ActiveModel::Serializer::Utils
+  module_function
+
+  def include_string_to_hash(included)
+    included.delete(' ').split(',').inject({}) do |hash, path|
+      hash.deep_merge!(path.split('.').reverse_each.inject({}) { |a, e| { e.to_sym => a } })
+    end
+  end
+
+  def include_args_to_hash(included)
+    case included
+    when Symbol
+      { included => {} }
+    when Hash
+      included.each_with_object({}) { |(key, value), hash| hash[key] = include_args_to_hash(value) }
```

Comment:
this aligns well with https://github.com/rails-api/active_model_serializers/pull/1127/files#diff-2e4bf15928c11c2f4eff9285340b602d

comments would be nice though. 

Have you tried seeing what happens if you somehow pass an invalid string as an include option?